specification/aodgv1/resource-manager/readme.python.md

```diff
@@ -0,0 +1,23 @@
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+
+``` yaml $(python)
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+package-name: azure-mgmt-orbital
```

Comment:
The name is already occupied by another service: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/orbital/resource-manager/readme.python.md?plain=1#L9